Actions/BuildALDoc/BuildALDoc.HelperFunctions.ps1

```diff
@@ -0,0 +1,238 @@
+﻿function RunningOnLinux {
+    $isPsCore = $PSVersionTable.PSVersion -ge "6.0.0"
+    return ($isPsCore -and $isLinux)
+}
+
+function FixBackslashes {
+    Param(
+        [string] $docfxPath
+    )
+
+    if (-not (RunningOnLinux)) {
+        return
+    }
+    $allFiles = @(get-childitem -path "$docfxPath/*" -Recurse -File | ForEach-Object { $_.FullName })
+    $allFiles | Where-Object { $_.Contains('\') } | ForEach-Object {
+        $newName = $_.Replace('\','/')
+        $folder = Split-Path -Path $newName -Parent
+        if (-not (Test-Path $folder -PathType Container)) {
+            New-Item -Path $folder -ItemType Directory | Out-Null
+        }
+        & cp $_ $newName
```

Comment:
This code is actually removed now, but the reason for using cp and rm was that under linux, a backslash is a perfectly normal character in a filename and aldoc executable ended up creating files with backslashes.
This function would (under linux) fix these files - and if I used Copy-Item and Remove-Item - it would revert to powershell functionality meaning that backslashes was folder seperators and then I couldn't fix the files.